c/src/ml-api-service-remote.c

```diff
@@ -366,27 +394,35 @@ _mlrs_process_remote_service (nns_edge_data_h data_h)
     case ML_REMOTE_SERVICE_TYPE_MODEL_URI:
     {
       GByteArray *array = g_byte_array_new ();
+      g_autofree gchar *dir_path = NULL;
 
       if (!_mlrs_get_data_from_uri ((gchar *) data, array)) {
         g_byte_array_free (array, TRUE);
         _ml_error_report_return (NNS_EDGE_ERROR_IO,
             "Failed to get data from uri: %s.", (gchar *) data);
       }
-      if (!_mlrs_model_register (service_key, data_h, array->data, array->len)) {
+
+      dir_path = _mlrs_get_model_dir_path (remote_s, service_key);
```

Comment:
_mlrs_get_model_dir_path() may return null in an ERROR.
How about returning an error if dir_path is NULL?

Reply:
This was resolved in previous PR. Thanks.